components/GridView.vue

```diff
@@ -131,29 +122,31 @@ function clearSelection(): void {
   window.getSelection()?.removeAllRanges()
 }
 
-onMounted(() => {
-  window.addEventListener('resize', resizeHandler)
-})
+function toggleEditMode(): void {
```

Comment:
Please change to arrow function. Also the one above.

Reply:
all "handlers" should be defined as arrow functions?
asking for future prs

Reply:
updated

Reply:
> all "handlers" should be defined as arrow functions? asking for future prs

yes please. We want to keep consistent code style across components.